spring-cloud-dataflow-tasklauncher/README.adoc

```diff
@@ -129,11 +129,11 @@ $curl http://localhost:9000 -H"Content-Type:application/json" -d '{"name":"times
 [source,bash]
 ----
 dataflow:>task execution list
-╔═════════╤══╤════════════════════════════╤════════════════════════════╤═════════╗
-║Task Name│ID│         Start Time         │          End Time          │Exit Code║
-╠═════════╪══╪════════════════════════════╪════════════════════════════╪═════════╣
-║timestamp│1 │Fri Aug 10 08:48:05 EDT 2018│Fri Aug 10 08:48:05 EDT 2018│0        ║
-╚═════════╧══╧════════════════════════════╧════════════════════════════╧═════════╝
+╔═══════════╤═══╤═══════════════════════════════╤═══════════════════════════════╤═══════════╗
```

Comment:
Did you check preview in Markdown editor in IDEA?

Reply:
Reverted to original block